tasks/data_handling/data_transformation.py

```diff
@@ -1,20 +1,24 @@
 import h5py
 import scipy as scp
 
-'''
-read_data(filename, mode)
-Function that transforms h5-file to a sparse matrix.
-
-filename: str. Path to h5-file to read. Default None.
-mode: str. 'csr' for Compressed Sparse Row or 'csc' for Compressed Sparse Column. Default 'csr'.
-
-Returns: sparse reconstructed matrix, tuples of form ((row, col), value).
-
-Written: ronjah@chalmers.se
-'''
-
-
-def read_data(filename: str = None, mode: str = 'csr') -> scp.sparse.csr_matrix:
+def read_data(filename: str = None, 
+              mode: str = 'csr'
+              ) -> scp.sparse.csr_matrix:
+    """
+    ### Function that transforms h5-file to a sparse matrix.
+
+    ---
+    
+    #### Args:
+        - filename (str): Path to h5-file to read. Default None.
+        - mode: str. 'csr' for Compressed Sparse Row or 'csc' for Compressed Sparse Column. Default 'csr'.
```

Comment:
Inconsistent formatting I guess we'll start enforcing `argument_name (type_name): description. Default value`